### ISSUE TITLE
Openning menus causes small freezing on video playback

### ISSUE THREAD

**Issue by vyscond** [feross/webtorrent-desktop#780]
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
`0.10.0 (0.96.0)`
**What operating system and version?**
`OS X El Captain 10.11.6 (15G31)`
**What did you do?**
Just openned some of the menus to check their options
**What did you expect to happen?**
Show the menus.
**What actually happened?**
The menus were showed but i noticed that this action causes a little freezing while some video is on playback.

**Comment by feross** [comment]
Yeah, I just confirmed this behavior. It's interesting that the audio playback is not affected, just the video.

I just tested another video app ([Playback](https://mafintosh.github.io/playback/) and it has the same behavior, so this is almost certainly a bug in Electron. We're not running any code when the menu is closed, so there's no way we could be blocking the event loop.

I opened this as an issue on Electron: https://github.com/electron/electron/issues/6897
